src/plugins/image_embeddable/public/components/image_viewer/image_viewer.tsx

```diff
@@ -100,7 +100,7 @@ export function ImageViewer({
               : undefined
           }
           loading={isScreenshotMode ? 'eager' : 'lazy'}
-          style={{
+          css={{
```

Comment:
It looks like `backgroundColor` is possibly dynamic, so I would suggest leaving part of this as `css` and pulling out `backgroundColor` to be an inline style